.github/workflows/test-package.yml

```
@@ -4,15 +4,14 @@ on: [ push, pull_request ]
 
 jobs:
   test:
-
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}-latest
     defaults:
       run:
         working-directory: 'go/src/github.com/bugsnag/bugsnag-go/v2' # relative to $GITHUB_WORKSPACE
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, windows-latest]
+        os: [ubuntu, windows]
```

Assuming you're expecting all the Ubuntu tests to fail at this stage, I would suggest changing it as follows:

```suggestion
        os: [windows]
        # Tests on Ubuntu skipped pending PLAT-XYZ 
        # os: [ubuntu, windows]
```

And have an explicit PLAT ticket to ensure we don't forget to reenable them.

---

done, added ticket PLAT-12608